Python/Base64_Encoder_Decoder/ConvertToBase64.py

```diff
@@ -0,0 +1,26 @@
+# This script is capable of converting some special/secret text
+# into 64 BASE Encoded String
+
+# importing base64
+
+def ConvertToBase64():
+	import base64
+
+	string = input("Enter string to encode:")
```

Comment:
Instead of taking CLI inputs, try to add Argument Parsers so that the input can be passed using Flags